### DISCUSSION TITLE
osu! wont load gamemodes

### DISCUSSION THREAD

**Discussion by darkcatsenpai** [ppy/osu#16101] [Q&A] [answered]
Whenever i try to add an gamemode (latest realease) it tells me
![Bild_2021-12-15_155209](https://user-images.githubusercontent.com/76266386/146208740-bf108ad0-e715-42c2-8b92-2585f8673f3a.png)
Is there Any way i can fix this? its happaned to me on every custom ruleset and i would really like to play them.

**Comment by weajus** [comment, marked as answer]
it's because they are not updated to lastest version of lazer, try updating the rulesets

**Reply by darkcatsenpai** [reply]
already tried that, im using the latest version for all of them. It could still be that they are outdated

**Reply by andy840119** [reply]
tell the author of ruleset owner to update lazer version.

**Reply by LumpBloom7** [reply]
> already tried that, im using the latest version for all of them. It could still be that they are outdated

No, from the screenshots, you aren't running the latest version of Hishigata. The latest version of Hishigata is [`2021.1127.0`](https://github.com/LumpBloom7/hishigata/releases/tag/2021.1127.0).

Do keep in mind that rulesets do not automatically update atm, unlike osu.
